src/ch/blinkenlights/android/vanilla/PlaybackService.java

```diff
@@ -2010,8 +2037,8 @@ public void onAudioFocusChange(int type)
 		Log.d("VanillaMusic", "audio focus change: " + type);
 		switch (type) {
 		case AudioManager.AUDIOFOCUS_LOSS_TRANSIENT_CAN_DUCK:
-			mDuckedLoss = (mState & FLAG_PLAYING) != 0;
-			unsetFlag(FLAG_PLAYING);
+			mDuckedLoss = true;
```

Comment:
Hmm, I'm looking at the various reads of `mState`, and they're inconsistently locked with `mStateLock`. I think I'll synchronise this read just to be safe, but do you think the non-locked reads are intentional?